examples/gallery/ui/pages/controls_page.slint

```diff
@@ -18,6 +18,21 @@ export component ControlsPage inherits Page {
             spacing: 8px;
             alignment: start;
 
+            Rectangle {  
+                width: 100px;
+                height: 20px;
+                background: green;
+
+                TouchArea {  
+                    width: 100%;
+                    height: 100%;
+                    clicked => { debug("clicked"); }
+                    double_clicked => {
```

Comment:
```suggestion
                    double-clicked => {
```

But I suppose this just a playground anyway. I suggest to use a dash for the docs though :)